File: PRAK404-2210817220019-DHEA_APRILINDA_UTAMI.c
#include <stdio.h>

int main()
{
    int input;
    float x,y, hasil;
    

    do{
    printf("Pilih Program\n");
    printf("1.Penjumlahan\n");
    printf("2.Pengurangan\n");
    printf("3.Perkalian\n");
    printf("4.Pembagian\n");
    printf("5.Exit\n");
    printf("masukkan nilai : ");
    scanf("%d", &input);
    if (input < 5){
        printf("masukkan nilai pertama : ");
        scanf("%f", &x);
        printf("Masukkan nilai kedua: ");
        scanf("%f", &y);
    }
    if (input == 1){
        hasil = x+y;
        printf("hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", x, y, hasil);
    }
     if (input == 2){
        hasil = x-y;
        printf("hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n",x, y, hasil);
    }
     if (input == 3){
        hasil = x*y;
        printf("hasil perkalian antara %.2f dengan %.2f adalah %.2f\n",x, y, hasil);
    }
      if (input == 4){
        hasil = x/y;
        printf("hasil pembagian antara %.2f dengan %.2f adalah %.2f\n",x, y, hasil);
    }   
     if (input > 5){
        printf("input anda salah, silahkan coba lagi\n");
    }
    }while (input != 5);
    printf("terimakasih, telah mengunakan kalkulator Dhea");

}

File: PRAK404-2210817220019-DHEA_APRILINDA_UTAMI.py
while True:
    print("Pilih Program")
    print("1.penjumlahan")
    print("2.pengurangan")
    print("3.perkalian")
    print("4.pembagian")
    print("5.exit")

    pilihan = int(input("masukkan Pilihan : "))
    if pilihan ==5:
        print("terimakasih, telah menggunakan kalkulator dhea")
        break
    if pilihan <5:
        a = int(input("masukkan angka pertama : "))
        b = int(input("masukkan angka kedua : "))
        if pilihan ==1:
                operasi = "penjumlahan"
                hasil = a+b
        if pilihan ==2:
                operasi = "pengurangan"
                hasil = a-b
        if pilihan ==3:
                operasi = "perkalian"
                hasil = a*b
        if pilihan ==4:
                operasi = "pembagian"
                hasil = a/b
        print("hasil",operasi,"antara %.2f dengan %.2f adalah %.2f" % (a,b,hasil))
        print("")
    if pilihan >5:
        print("input anda salah, silahkan coba lagi")
        print("")